### PR TITLE
Basic Functionality

### PR DESCRIPTION
**Pull request by MarcelSchueren**
Basic functionality with db, frontend, backend. User can store some stocks in the db and the app updates their prices. Tests work, but CI doesn't, maybe because I refactored some directories a couple of days ago. CI will maybe work after pull-request is merged.

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
-  );
+    const {stocks} = useStocks()
+    return (
+        <div>
+            <Router>
+                <CssBaseline/>
+                <Header/>
+                <Switch>
+                    <Route path="/new">
+                        <NewDepotPage/>
```
Comment (larsschloegel) [inline frontend/src/App.js:22]: i think the page name is better without "Page" at the end -> Clean Code Explained (Avoid Disinformation)

File: frontend/src/components/NavigationBar.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    const [value, setValue] = useState('');
+    let history = useHistory();
+
+    const handleChange = (event, newValue) => {
+        setValue(newValue);
+        history.push(newValue);
+    };
+
+    return (
+        <Paper sx={{position: 'fixed', bottom: 0, left: 0, right: 0}} elevation={3}>
```
Comment (larsschloegel) [inline frontend/src/components/NavigationBar.js:21]: i would add padding:0. Try it ;)

File: frontend/src/pages/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import {Typography} from "@mui/material";
+import PortfolioItem from "../components/PortfolioItem";
+
+export default function HomePage({stocks}) {
+
+    return (
+        <div>
+            <Typography variant="h3">HomePage</Typography>
+            {stocks.map(stock => {
+                return <PortfolioItem stock={stock} key={stock.id}/>
```
Comment (larsschloegel) [inline frontend/src/pages/HomePage.js:10]: double return? it think you can write 
           {
                stocks.map(stock => <PortfolioItem stock={stock} key={stock.id}/>)
            }

**Review by larsschloegel — COMMENTED** [review]
Good job marcel ;).

**Review by larsschloegel — APPROVED** [review]

File: frontend/src/App.js (hunk 1 above)
Comment (MarcelSchueren) [reply frontend/src/App.js:22]: I will rethink the hole passage with my pages and the routes, thanks for your inspiration.

File: frontend/src/components/NavigationBar.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (MarcelSchueren) [reply frontend/src/components/NavigationBar.js:21]: I copy pasted it to fix the element.... will rethink this, thanks.

**Pull request merged**